dev/devicelab/lib/versions/gallery.dart

```diff
@@ -0,0 +1,6 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The pinned version of flutter gallery, used for devicelab tests.
+const String galleryVersion = 'a208eac6e6e8336ae9820e54c572c099231f1da2';
```

Comment:
This is awesome. Your PR LGTM.

My only question is if there is an official documented procedure for rolling the gallery here, and if it can be updated to reference this file.

Reply:
I don't know. I'm asking right now.

Reply:
+1, interested in how this works as gallery changes - don't we want master to test against the newest gallery?  this seems more reasonable for branches than master (but I may be misunderstanding)

Reply:
If we always ask `flutter/flutter:master` to clone `flutter/gallery:master`, an error in Gallery might cause Flutter's build to fail. These tests always uses latest `flutter/flutter:master`.
I think the best way is to use a commit hash instead of a branch, and set up a process to roll it regularly. (But this is a future task.)

Reply:
@christopherfujino @pcsosinski 

There isn't a process to roll the gallery version yet. It would be a good idea to add this process, some time in the future.